client/lib/feed-post-store/index.js

```diff
@@ -83,7 +83,7 @@ FeedPostStore.dispatchToken = Dispatcher.register( function( payload ) {
 		case FeedPostActionType.RECEIVE_FEED_POST:
 			if ( action.error ) {
 				const error = {
-					status_code: -1,
+					status_code: action.error.statusCode ? action.error.statusCode : -1,
```

Comment:
Just being cautious here. It looks from [receiveError](https://github.com/Automattic/wp-calypso/blob/5f44fdb50e31aa54161fa8ad96cabd6363c7fb98/client/lib/feed-post-store/index.js#L235) that not finding a status code is a possibility.